user_sync/app.py

```diff
@@ -263,6 +263,9 @@ def log_parameters(argv, config_loader):
     :type config_loader: user_sync.config.ConfigLoader
     :return: None
     """
+    python_version = ' ' * 13 + 'Python %s.%s.%s' % sys.version_info[:3]
```

Comment:
There's no point of prefixing with spaces, and we should put out the OS type as well as the Python version.  So let's go with
```python
logger.info('Python version: %s.%s.%s on %s' % (sys.version_info[:3] + (sys.platform,)))
```